cli/src/main/scala/scala/scalanative/cli/options/MiscOptions.scala

```diff
@@ -0,0 +1,10 @@
+package scala.scalanative.cli.options
+
+import caseapp.HelpMessage
+import caseapp.Group
+
+case class MiscOptions(
+    @Group("Help")
+    @HelpMessage("Print scala-native-cli version and exit")
+    val version: Boolean = false
```

Comment:
Cannot this be handled by CaseApp? I think there is a `AppVersion` but I might be wrong annotation

Reply:
I did not get it to work, in saw that in scala-cli a separate command is used, and in CaseApp sources it seems to be ignored. I can move that additional parsing logic out from main, if necessary

Reply:
I also personally like having the --version option, rather than relying on --help for that.

Reply:
Ok, good then. I was hoping that such a feature would be included.